site/docs/features/management.md

```diff
@@ -5,34 +5,107 @@ service can be scheduled and Nessie reports the outcome of each scheduled operat
 Scheduled operations require that Nessie have access to a Spark cluster to complete
 those operations and many of them are distributed compute operations.
 
-!!! info
-    Table management services are currently in progress and are not yet included in a released version of Nessie.
-
 ## Garbage Collection
 
-Since Nessie is maintaining many different versions of metadata and data-pointers simultaneously,
+Since Nessie is maintaining many versions of metadata and data-pointers simultaneously,
 you must rely on Nessie to clean up old data. Nessie calls this garbage collection.
 
-Garbage collection policies are configurable and can be either destructive or instructive.
-To configure garbage collection, you must set the following settings:
-
-### Collection Types
+There are at least two steps to a garbage collection action. The first steps are instructive, and the last step is destructive.
 
-|Collection|Modes|Notes|
-|-|-|-|
-|Unreferenced Data|Instructive or Destructive|Deletes any objects that haven't been **directly** referenced by a tag or branch for 30 days.|
-|Unreferenced Commit|Deletes internal Nessie objects that are no longer referenced|Destructive|
-|Unreferenced Metadata|Deletes metadata objects that are no longer referenced[^1]|
```

Comment:
Think you can remove the footnote then as well.

Reply:
good catch, gone